### ISSUE TITLE
Built-in accessibility checker

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/roadmap#19]
## Summary

As part of our goal to conform with [ATAG 2.0](https://www.w3.org/TR/ATAG20/), we need a built-in accessibility checker for Wagtail. We’ve currently settled on [Sa11y](https://sa11y.netlify.app/) as a good candidate. It’s based on Tota11y, similarly to the popular [wagtail-accessibility](https://github.com/neon-jungle/wagtail-accessibility) package, but is purpose-built for content management use cases.

## Intended Outcome

This kind of testing tool should be built into the CMS.

## How will it work?

Sa11y will be available in Wagtail’s full page previews, and within the page editor, either in the live preview panel, or in a separate panel alongside it. It will be configurable by site owners and CMS users.

## More information

- See [Built-in accessibility checker for Wagtail #9262](https://github.com/wagtail/wagtail/discussions/9262).
